### ISSUE TITLE
这个报错怎么整

### ISSUE THREAD

**Issue by 709411fan** [RVC-Boss/GPT-SoVITS#208]
py:139: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`.

**Comment by BlacknoSheep** [comment]
这是警告不是报错，不用管。

**Comment by 709411fan** [comment]
但是到这停了，运行不了

**Comment by 709411fan** [comment]
skipped_phone:  0 , skipped_dur:  0
total left:  100
ssl_proj.weight not requires_grad
ssl_proj.bias not requires_grad
INFO:cf:loaded pretrained GPT_SoVITS/pretrained_models/s2G488k.pth
<All keys matched successfully>
INFO:cf:loaded pretrained GPT_SoVITS/pretrained_models/s2D488k.pth
<All keys matched successfully>
E:\kelongyuyin\GPT-SoVITS\runtime\lib\site-packages\torch\optim\lr_scheduler.py:139: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "

**Comment by RVC-Boss** [comment]
win10?

**Comment by 709411fan** [comment]
好了，我内存的原因，换了个内存条就没事了
